.travis.yml

```diff
@@ -5,6 +5,7 @@ php:
     - 5.6
     - 5.5
     - 5.4
+    - 5.3.3
```

Comment:
The point of this was to prove that it works on php 5.3.3 and not just a much later version of 5.3 though isn't it?